### PR TITLE
[amazondashbutton] Update README.md

### PR DESCRIPTION
**Pull request by ThomDietrich**
Markdown fixup
Phrasing
Some links
Windows installation hints
foo

Signed-off-by: Thomas Dietrich <thomas.dietrich@tu-ilmenau.de> (github: ThomDietrich)

### CODE REVIEW

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 1]
```diff
@@ -1,87 +1,106 @@
...
+Instructions for Debian/Ubuntu are given below.
 
-Source: [https://www.pcap4j.org/](https://www.pcap4j.org/)
+### Installing libpcap on Debian/Ubuntu
 
-#### Debian/Ubuntu
+Installing [libpcap](http://www.tcpdump.org/) should be as simple as:
 
 ```shell
 apt-get install libpcap-dev
```
Comment (BClark09) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: sudo?

**Comment by ThomDietrich** [comment]
Done, Done and Done. I've also rephrased the last part while I was reading it again. feel free to read it again.

On a side note: I do not yet own a Dash Button. I've ordered two today, wanted to inform myself in the documentation prior to that and couldn't hold my hands still :sweat_smile: 

Would you be able to elaborate on the "Restrict Internet" topic? I've added "Otherwise the Dash Button will always send notifications." which I found in the community forum. What does it mean exactly? We should answer the question if one can use the button without restricting the internet connection. If that is not the case, we should warn about this restriction right in the introduction.

**Comment by BClark09** [comment]
Ah, that's probably referring to the fact that the button would connect to the Amazon servers with each press. Since you haven't set anything up, you get a push notification on your phone asking you to set it up with each press. 

Disconnecting it from the Internet is good for closed systems, but it'll leave the device searching for much longer. You'll curb the battery life this way.

**Comment by ThomDietrich** [comment]
> You'll curb the battery life this way.

Aha! That needs to go into the README :)

> push notification on your phone

What if I install the app or block notifications for it?


I think that's an important issue because some might not be able to simply block internet access for devices. I'm honestly not sure if the Unifi system at my parents place supports it...

**Comment by BClark09** [comment]
Agreed, it's inconsequential right now. If you block the notifications on your phone then I don't think else happens! I imagine some people would want to be mindful that amazon could setup a default order with the card you used to buy later models of the dash button, but since that's not the case now...

**Comment by OLibutzki** [comment]
@ThomDietrich As @BClark09 mentioned the internet access should be blocked because otherwise you contact the Amazon servers with every button press.

There seems to be no other effect than getting a push notification by the amazon app stating that your Dash button is not configured so far. Blocking this notification might be sufficient, but you have to be aware that Amazon is notified each time you press your button.

It's likely that they detect improper use of your Dash button as no one would press a button that often without triggering a purchase.

In my opinion reasonable to describe both ways (Firewall blocking rule and blocking Amazon app notifications) with their (dis-)advantages.

**Comment by ThomDietrich** [comment]
Hey guys! Thanks for your help in the matter :beer: I've pushed another change.

Maintainers: I'm finished :v:

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 2]
```diff
@@ -1,87 +1,120 @@
...
+Be aware:
+Every time a Dash Button is pressed a request will be sent to the Amazon servers.
+If no product was configured for the Button, a notification will be presented by the Amazon app on your smartphone.
 
+To prevent the Dash Button from contacting the Amazon Servers, block Internet access for the device.
+Please refer to the documentation of your network's router for details.
+If your network doesn't provide that option, you can at least deal with the notifications by either uninstalling the Amazon app or disabling notifications for it (possible on most smartphone OSs).
 
-## Supported Things
+Preventing the communication with the Amazon servers or the Amazon app is **not** necessary to integrate the Dash Button in openHAB.
```
Comment (BClark09) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: You mentioned that we should warn others of the battery usage when disconnecting it. Might be worth adding that statement here?

Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: Ah thanks! Totally missed that.

**Comment by martinvw** [comment]
> Maintainers: I'm finished ✌️

Almost 😉

**Comment by ThomDietrich** [comment]
finished²

**Comment by martinvw** [comment]
The first 2.2 or which is merged, thanks

**Pull request merged**